basic-sql-commands.md

```diff
@@ -98,14 +98,14 @@ To verify that all columns were created correctly, use the [`SHOW COLUMNS FROM`]
 SHOW COLUMNS FROM table1;
 ```
 ```
-+----------+--------+-------+---------------------------+
-|  Field   |  Type  | Null  |          Default          |
-+----------+--------+-------+---------------------------+
-| column_a | INT    | true  | NULL                      |
-| column_b | FLOAT  | true  | NULL                      |
-| column_c | DATE   | true  | NULL                      |
-| column_d | STRING | true  | NULL                      |
-+----------+--------+-------+---------------------------+
++------------+----------+-------+---------+
+|   Field    |   Type   | Null  | Default |
++------------+----------+-------+---------+
+| "column_a" | "INT"    | false | NULL    |
```

Comment:
DB names, table names, and types didn't use to show up with quotes, but they do now, so I'm just trying to make the docs consistent. But if quotes shouldn't be used in these cases, I can open a new ticket...